source/graphql/schema/toschemafile.d

```diff
@@ -158,15 +158,15 @@ Visibility traceType(GQLDType t, ref TraceableType[string] tab) {
 	return tab[map.name].vis;
 }
 
-string gqldTypeToString(const(GQLDType) t, string nameSuffix = "") {
+string gqldTypeToString(const(GQLDType) t, string nameSuffix = "", bool nonNullable = true) {
```

Comment:
could you please turn the bool into a std.typecons.Flag!"NonNullable" passing boolean around is not nice.

I properly do it somewhere as well but, consider this a "do as I say, not as I do" situtation.